pages/c/plans/[id].js

```diff
@@ -34,12 +33,13 @@ const CustomerPlanSummary = ({ planId, initialPlan, token }) => {
 
 CustomerPlanSummary.getInitialProps = async ({ query: { id }, req, res }) => {
   try {
-    const token = createToken();
+    const token = createToken('');
```

Comment:
Calling this with no arguments should also be fine, I think `undefined` should result in "name" not appearing in the token at all - which is probably more the intent we're after than an empty string being set for name?